src/items/itemDialog.component.tsx

```diff
@@ -0,0 +1,632 @@
+                                propertyErrors[index]
+                                  ? // If 'propertyErrors[index]' is truthy, perform the following checks:
+                                    property.mandatory && !propertyValues[index]
+                                    ? // If 'property' is mandatory and 'propertyValues[index]' is empty, return a mandatory field error message
+                                      'This field is mandatory'
+                                    : property.type === 'number' &&
+                                      isNaN(Number(propertyValues[index])) &&
+                                      'Please enter a valid number' // If 'property' is of type 'number' and 'propertyValues[index]' is not a valid number, return an invalid number error message
+                                  : // If 'propertyErrors[index]' is falsy, return an empty string (no error)
+                                    ''
+                              }
+                            />
+                          )}
+                        </Grid>
+                        <Grid item xs={1} sx={{ display: 'flex' }}>
+                          <Tooltip
+                            sx={{ alignItems: 'center' }}
+                            title={
+                              <div>
+                                <Typography>Name: {property.name}</Typography>
+                                <Typography>Unit: {property.unit}</Typography>
+                                <Typography>
+                                  Type:{' '}
+                                  {property.type === 'string'
+                                    ? 'text'
+                                    : property.type}
+                                </Typography>
+                              </div>
+                            }
+                            placement="right"
+                            enterTouchDelay={0}
+                          >
+                            <IconButton size="small">
+                              <InfoOutlinedIcon />
+                            </IconButton>
+                          </Tooltip>
+                        </Grid>
+                      </Grid>
+                    </Grid>
+                  )
+                )}
+              </Grid>
+            )}
+          </Grid>
+        </Grid>
+      </DialogContent>
+      <DialogActions sx={{ flexDirection: 'column', padding: '0px 24px' }}>
+        <Box
+          sx={{ display: 'flex', alignItems: 'center', width: '100%' }}
+        ></Box>
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'center',
+            width: '100%',
+            my: 2,
+          }}
+        >
+          <Button
+            variant="outlined"
+            sx={{ width: '50%', mx: 1 }}
+            onClick={handleClose}
+          >
+            Cancel
+          </Button>
+          <Button
```

Comment:
add logic so save button gets disabled when there is an error in the dialog. see #189